app/(dashboard)/layout.tsx

```
@@ -29,15 +36,35 @@ export const viewport = {
-    <NuqsAdapter>
-      <Toaster richColors />
-      <SentryContext />
-      <TRPCReactProvider>
-        <StandaloneDisplayIntegration />
-        <HydrateClient>{children}</HydrateClient>
-      </TRPCReactProvider>
-    </NuqsAdapter>
-  );
+export default async function RootLayout({ children }: { children: React.ReactNode }) {
+  try {
+    const { preferences } = await api.mailbox.get();
```

ESLint

[error] 'preferences' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)